js/ui/motif.reveal.es6

```diff
@@ -0,0 +1,778 @@
+/*!
+ * Motif Reveal v2.3.0 (2018-6-15)
+ * Show and hide things with class(es)
+ *
+ * Reveal accepts single or multiple IDs of target elements and, on click or 
+ * hover, adds and removes classes from those target elements as well as the 
+ * triggering element. It can group "reveals" so that you can dictate whether 
+ * multiple targets can be active at the same time.
+ *
+ * Reveal keeps track of all these triggers and targets and states by creating
+ * a "reference" in a `window.Reveal` object. You can see a model of that
+ * reference at the bottom of this file. This allows us to be able to track
+ * and manipulate these reveals outside of this plugin. In future development,
+ * we will also be able to guarantee that targets of multiple reveals and 
+ * groups will retain their appropriate state.
+ * 
+ * http://getmotif.com
+ * 
+ * @author Jonathan Pacheco <jonathan@lifeblue.com>
+ */
+
+const DEFAULTS = {
```

Comment:
All of these constants could go into the Reveal object, especially if they are specific to the class, and thus not used elsewhere, instead of putting them into the global scope.

Reply:
Thanks for the heads up. My understanding is that since `motif.reveal.es6.js` is considered a Module (it has an `export`), every `let` and `const`, even at the "top level" of the file, is scoped to this file (whether it's through Babel's transpiling or in native browser implementation of Modules).

This chart helps: https://twitter.com/kangax/status/567330097603284992

In that case, it becomes more of a stylistic preference, I think: treat the file as if it's self contained and take advantage of generic names, or still scope by name/convention.

Additionally, I think I chose to separate it out because I didn't really see a case where the creator of a `Reveal` object would want access to the default settings. (Though, in this initial pass, I did not take the time to apply that same logic to any of the other functions that _are_ attached to the `Reveal` class. 😄)

(Side tangent: if I'm reading that chart correctly, a `var` *would* be put into the global scope, though the way Babel wraps every individual module in a function, I'm not positive it always exposes `var`s globally, or if it even has to.)

Reply:
Yes, every imported file is contained in an iife; none of those variables (which are not explicitly attached to `window`) would be exposed outside of the imported module iife.

I should have clarified further when I said `global` meaning, they have more visibility by more local classes than is necessary. And, as a matter of stylistic preference, it is mine to prefer to not give any more access to data than required, since they are default values, and constants at that... hence "...these constants _could_...".

Kudos on moving things to es6.

Reply:
Ah, I see what you mean. I needed to jog my (poor) memory and look at the file again. Usually I write these things with a single `class` per file and put utility functions in separate files, but here, since I've only updated one file and not the whole JS lot, I lumped a couple of classes and a utility function into one file. I now see your point, all those constants apply to one class or the other, not necessarily to everything in this file, so there's the opportunity to scope them further.